### PR TITLE
kops: Add a PR builder for the kops repo as well

### PR DESCRIPTION
**Pull request by zmerlynn**
This is more or less duplicative of https://github.com/kubernetes/test-infra/blob/master/jobs/pull-kubernetes-e2e-kops-aws.sh, but I didn't have a great way to dedupe the code yet.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1162)

### CODE REVIEW

**Comment by spxtr** [comment]
If this works then it's the first non-k8s repo testing k8s directly. That'll be interesting.

/lgtm

File: jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-pull.yaml [hunk 1]
```diff
@@ -61,6 +61,10 @@
         job-name: pull-heapster-e2e
         max-total: 5
         repo-name: 'k8s.io/heapster'
+    - kops-e2e-kubernetes-aws:  # owner: zml@google.com
+        job-name: pull-kops-e2e-kubernetes-aws
+        max-total: 5
+        repo-name: 'k8s.io/heapster'
```
Comment (ixdy) [inline jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-pull.yaml]: is this correct?

Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-pull.yaml]: Ha. No.

**Comment by zmerlynn** [comment]
@ixdy: PTAL

**Comment by fejta** [comment]
Joe/Jeff please add me back if I'm needed.

**Comment by zmerlynn** [comment]
I'm about to merge this, no worries.

**Pull request merged**